eng/BuildTask.Packages.props

```
@@ -0,0 +1,9 @@
+<Project>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net472'">
```

This be conditioned on the TFM family not the exact TFM:

```suggestion
  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
```